CanonicalLtd/landscape-client
# Replace usermod calls by PAM to remove passlib (LP: #1743558)

Testing instructions:

* build and install package
  * check package does not depend on python3-passlib (universe)
* register client
* go to computer/users tab
  * create a new user
  * edit an existing user password
  * edit root password
* verify passwords changed correctly
